manifests/charts/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -214,6 +214,10 @@ spec:
           command:
           - pilot-agent
           - wait
+          {{ if (isset .ObjectMeta.Annotations `sidecar.istio.io/holdApplicationTimeoutSeconds`) -}}
```

Comment:
is `sidecar.istio.io/holdApplicationTimeoutSeconds` a documented annotation?

Reply:
No. it is not yet. if this PR is accepted, I will add this to the doc page as well.

Reply:
i think you should raise api PR first.